### DISCUSSION TITLE
Pusblish Start & End Date - Dead form field in admin if only one is used

### DISCUSSION THREAD

**Discussion by madsem** [area17/twill#999] [Ideas]
According to the comments in the migrations, you can use only `$table->timestamp('publish_start_date')->nullable();` if you don't need an `publish_end_date`.

The result is that the admin form still displays the end date though, but it's just a dead form field that cannot be saved.

It's not really a bug, more cosmetical, but I think it should still be reported so it can be polished :)

**Comment by ifox** [comment]
Hey @madsem!

You are perfectly right to flag those issues. This is what is going to make Twill really shine as we're really committed to providing the best user experience. So what you report here is definitely helpful, thank you!
